### PR TITLE
Fix invalid PHP code samples in utests

### PR DESCRIPTION
**Pull request by SpacePossum**
next bunch of test data fixes, @ see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1538

### CODE REVIEW

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Comment by keradus** [comment]
Haha, it;'s not good to commit during merging :D But yeah, you can't know

**Comment by keradus** [comment]
OK, first commit was grabbed into merge, next 2 didn't. 1.11 is green. For 2 latest commits please open new PR.

**Comment by SpacePossum** [comment]
haha oh, sorry @keradus :)
I'll continue on a new branch and PR :+1:

File: Symfony/CS/Tests/Fixer/Contrib/LongArraySyntaxFixerTest.php [hunk 1]
```diff
@@ -19,6 +19,25 @@
...
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideSkipExamples()
+    {
+        return array(
+            array('<?php $x[2] = 1;'),
+            array('<?php $x["a"] = 1;'),
+            array('<?php $x[] = 1;'),
+            array('<?php $x[ ] = 1;'),
```
Comment (stof) [inline Symfony/CS/Tests/Fixer/Contrib/LongArraySyntaxFixerTest.php]: do these actually require PHP 5.4+ ?

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Contrib/LongArraySyntaxFixerTest.php]: To be fixed? don't think so, but I'm not sure, for the tests to pass yes it is needed.
We are taking another route now so I didn't change any other short array test code until we figured out what to do, more discussing is in this PR:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1563#discussion_r46339790

**Pull request closed without merging**